### PR TITLE
RHIROS-799 fixed getting started URL

### PR DESCRIPTION
**Pull request by PreetiW**
### Description 
updated getting stated URL as per discussion [here](https://ansible.slack.com/archives/C01C1GVT3TP/p1662534998181449?thread_ts=1662456323.450929&cid=C01C1GVT3TP)

### Jira
https://issues.redhat.com/browse/RHIROS-799

### Screenshot

image

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ export const NO_DATA_VALUE = 'N/A';
 
 // Getting started URL
 // eslint-disable-next-line max-len
-export const GETTING_STARTED_URL = 'https://access.redhat.com/documentation/en-us/red_hat_insights/2022/html/assessing_and_monitoring_rhel_resource_optimization_with_insights_for_red_hat_enterprise_linux/proc-ros-install_ros-getting-started';
+export const GETTING_STARTED_URL = 'https://access.redhat.com/documentation/en-us/red_hat_insights/2022/html/assessing_and_monitoring_rhel_resource_optimization_with_insights_for_red_hat_enterprise_linux/proc-ansible-install_ros-install#doc-wrapper';
```
Comment (kgaikwad) [inline src/constants.js:25]: Observation - previous URL was pointing to getting started section. Why can't we pointing to getting started section with modified URL? 
i.e https://access.redhat.com/documentation/en-us/red_hat_insights/2022/html/assessing_and_monitoring_rhel_resource_optimization_with_insights_for_red_hat_enterprise_linux/con-ros-overview_ros-overview

**Review by kgaikwad — CHANGES_REQUESTED** [review]

File: src/constants.js (hunk 1 above)
Comment (PreetiW) [reply src/constants.js:25]: good catch @kgaikwad let me confirm with Pau and get back to you!

Comment (PreetiW) [reply src/constants.js]: discussed with Pau and team and updated the URL to above @kgaikwad

**Review by kgaikwad — APPROVED** [review]
Legit :+1:

**Pull request merged**